opengrok-indexer/src/main/java/org/opengrok/indexer/history/PerforceHistoryParser.java

```diff
@@ -87,7 +87,7 @@ public static History getRevisions(File file, String rev) throws IOException {
         cmd.add("p4");
         cmd.add("filelog");
         cmd.add("-lti");
-        cmd.add(file.getName() + PerforceRepository.getRevisionCmd(rev));
+        cmd.add(PerforceRepository.protectPerforceFilename(file.getName()) + PerforceRepository.getRevisionCmd(rev));
```

Comment:
import and use the method directly

Reply:
I'm sorry, I don't understand what you're asking for here.  This is modeled after the other method calls to methods of PerforceRepository, including PerforceRepository.getRevisionCmd in this same line.  Can you clarify what you'd like to see?

Reply:
If you add
```java
import static org.opengrok.indexer.history.PerforceRepository.protectPerforceFilename;
```
then you can replace the line with just:
```java
            String prot = protectPerforceFilename(ent.getKey());
```

Reply:
same thing as with the test method - import the static method and use it directly without having to specify the class.

Reply:
Okay.  Can do.  I was just following the example of the calls to `getRevisionCmd`, but happy to do it this way instead.  Thanks for explaining how to accomplish it.